### PR TITLE
Create servlet class

### PR DESCRIPTION
**Pull request by tinotenda6**
Created a servlet class, which prints HelloWorld when url "/index1.html" is invoked.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/HardCodedStringServlet.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns HTML that contains the page view count. */
```
Comment (zhaonian) [inline portfolio/src/main/java/com/google/sps/servlets/HardCodedStringServlet.java]: Could you update the doc please?

File: portfolio/src/main/java/com/google/sps/servlets/HardCodedStringServlet.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+@WebServlet("/index1.html")
```
Comment (zhaonian) [inline portfolio/src/main/java/com/google/sps/servlets/HardCodedStringServlet.java]: not blocking. 
how about "/index.html" or a more meaningful name?

**Review by zhaonian — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/HardCodedStringServlet.java (hunk 1 above)
Comment (tinotenda6) [reply portfolio/src/main/java/com/google/sps/servlets/HardCodedStringServlet.java]: Yes, thanks!

File: portfolio/src/main/java/com/google/sps/servlets/HardCodedStringServlet.java (hunk 2 above)
Comment (tinotenda6) [reply portfolio/src/main/java/com/google/sps/servlets/HardCodedStringServlet.java]: Oh okay, I changed to mypage.html.

**Pull request merged**